src/command_modules/azure-cli-container/HISTORY.rst

```diff
@@ -3,9 +3,7 @@
 Release History
 ===============
 
-0.3.14
-++++++
-* Minor fixes.
```

Comment:
Why are you making changes to this module?

Reply:
the file looks like the one in master, this happened when I rebased.